### ISSUE TITLE
JS Callbacks

### ISSUE THREAD

**Issue by wjbmattingly** [wjbmattingly/bulk-image-clustering#2]
@joelsjlee I think we should consider adding JS Callbacks instead of Python call backs. We could let users save the entire server as an HTML file instead of curdoc(). This would allow the HTML file to be shared and used independent of Python. Thoughts? I don't have experience writing custom JS Callbacks in Bokeh and not sure about the time it would take, but now that everything is updated via the same update function, I would imagine that the process would be a bit more straightforward.

**Comment by joelsjlee** [comment]
I agree that we should at least take a shot at making all of the functionality of the visualization up to this point possible through JS callbacks. I'll try and make some time to work this out. 

And yes, making this whole visualization available as an HTML file would be nice in a scenario where we can generate the HTML file and then say, pass it to a web app to be served.

**Comment by wjbmattingly** [comment]
Awesome! Thanks!
